cli/src/commands/v2-migrations/20171127214900_migrate_identifiers.js

```diff
@@ -47,8 +48,14 @@ const updateIdentifierFields = async () => {
 };
 
 const cloneIdentifiersToNewCollection = async () => {
-  const pipeline = [{ $match: {} }, { $out: newIdentsCollectionName }];
-  await connection.collection(oldIdentsCollectionName).aggregate(pipeline);
+  const pipeline = [{ $match: {} }];
+  await new Promise((resolve, reject) => {
+    connection.collection(oldIdentsCollectionName).aggregate(pipeline, { out: newIdentsCollectionName }, (err) => {
```

Comment:
Didn't actually need to wrap this I don't think, should be able to use `.catch`.